client/src/pages/Home/HeroBanner/ChatBox.tsx

```diff
@@ -0,0 +1,38 @@
+export default function ChatBox({
+  text,
+  bgColor,
+  textColor,
+  borderRadius,
+  top,
+  bottom,
+  left,
+  right,
+}: {
+  text: string;
+  bgColor: string;
+  textColor: string;
+  borderRadius: string;
+  top?: string;
+  bottom?: string;
+  left?: string;
+  right?: string;
+}) {
+  const styleAttributes: { [key: string]: string } = {};
+  if (top) styleAttributes.top = top;
```

Comment:
May refactor the code to something similar to this :

const styleAttributes = {
  ...(top && { top }),
  ...(bottom && { bottom }),
  ...(left && { left }),
  ...(right && { right }),
  backgroundColor: bgColor,
  color: textColor,
  'border-radius': borderRadius
};